src/lib/support/logging/CHIPLogging.cpp

```diff
@@ -100,8 +100,8 @@ const char ModuleNames[] = "-\0\0" // None
                            "SD\0"  // ServiceDirectory
                            "SP\0"  // ServiceProvisioning
                            "SWU"   // SoftwareUpdate
-                           "TP\0"  // TokenPairing
-                           "TS\0"  // TimeService
+                           "FS\0"  // FailSafe
+                           "TS\0"  // TimeServices
```

Comment:
I had changed this comment from TimeServices to TimeService to match `kLogModule_TimeService`.  I don't have an opinion on which is better, but changing this comment is a smaller delta.

```suggestion
                           "TS\0"  // TimeService
```